### PR TITLE
Read RRFS effective radii

### LINKED ISSUE #670: Read in effective radii for GOES simulated brightness temps in RRFS
This small change (from Will Lewis at Wisconsin) reads in effective radii from FV3 in order to use them in calculations of GOES simulated brightness temperature output.

### PR DESCRIPTION
**Pull request by EricJames-NOAA**
This PR allows reading in the effective radii variables from RRFS, which are used in the calculation of GOES simulated brightness temperatures.  

There are no additional output variables, so no changes to the XML files are needed.  The new code slightly changes the output for the GOES simulated brightness temperature variables.  

The code was tested for the RRFS_CONUS_3km system on Jet.  The code was developed by Will Lewis (Univ of Wisconsin).

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA Can you sync your branch with the latest UPP develop?

File: sorc/ncep_post.fd/INITPOST_NETCDF.f [hunk 1]
```diff
@@ -920,6 +922,21 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
 !          ,cfr(isa,jsa,l),isa,jsa,l
 !      enddo
 
+!     WL add cieffr for Thompson scheme cloud ice effective radius
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/INITPOST_NETCDF.f:925]: @EricJames-NOAA Do you know if these new radii will be  as default model output when the UFS model uses Thompson scheme, or just for RRFS?

Comment (EricJames-NOAA) [reply sorc/ncep_post.fd/INITPOST_NETCDF.f:925]: @WenMeng-NOAA we are not sure about that.  We might need to reach out to the model developers to answer your question.  However, we did not recently make any modifications to the RRFS diag table to output these radii; they were already available in the netCDF files.

Comment (WenMeng-NOAA) [reply sorc/ncep_post.fd/INITPOST_NETCDF.f:925]: @EricJames-NOAA Okay. We may keep the current setting for reading them from all UFS based models.

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA Do you have sample RRFS model files including effective radii for my testing?

**Comment by EricJames-NOAA** [comment]
> @EricJames-NOAA Do you have sample RRFS model files including effective radii for my testing?

@WenMeng-NOAA I have test files here on Jet:

/lfs4/BMC/wrfruc/ejames/RRFS/stmp/tmpnwprd/RRFS_conus_3km/2023030915/fcst_fv3lam

Let me know if you need them on Hera.  Thanks.

**Comment by WenMeng-NOAA** [comment]
> > @EricJames-NOAA Do you have sample RRFS model files including effective radii for my testing?
> 
> @WenMeng-NOAA I have test files here on Jet:
> 
> /lfs4/BMC/wrfruc/ejames/RRFS/stmp/tmpnwprd/RRFS_conus_3km/2023030915/fcst_fv3lam
> 
> Let me know if you need them on Hera. Thanks.

@EricJames-NOAA It would be great if you can transfer them to Hera. Thanks!

**Comment by EricJames-NOAA** [comment]
> 

OK, I copied them to /scratch1/BMC/wrfruc/ejames/wen_radii/

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA I ran two tests with your branch and develop branch pn Hera.
```
your branch: /scratch1/NCEPDEV/stmp2/Wen.Meng/rrfs_2023030915-after
develop branch: /scratch1/NCEPDEV/stmp2/Wen.Meng/rrfs_2023030915-before
```
I can see  changes in simulated BT fields in NATLEV dataset between two tests:
```
1529:1404731937:SBTA167:top of atmosphere:rpn_corr=0.831499:rpn_rms=8.65094
1530:1407483266:SBTA168:top of atmosphere:rpn_corr=0.995889:rpn_rms=0.503038
1531:1409196345:SBTA169:top of atmosphere:rpn_corr=0.991953:rpn_rms=0.924436
1532:1411015057:SBTA1610:top of atmosphere:rpn_corr=0.989674:rpn_rms=1.33921
1533:1413024108:SBTA1611:top of atmosphere:rpn_corr=0.992438:rpn_rms=2.07622
1534:1415614858:SBTA1612:top of atmosphere:rpn_corr=0.998844:rpn_rms=0.724649
1535:1417981775:SBTA1613:top of atmosphere:rpn_corr=0.997619:rpn_rms=1.21845
1536:1420618317:SBTA1614:top of atmosphere:rpn_corr=0.994744:rpn_rms=1.83284
1537:1423273914:SBTA1615:top of atmosphere:rpn_corr=0.990948:rpn_rms=2.36515
1538:1425904189:SBTA1616:top of atmosphere:rpn_corr=0.990229:rpn_rms=1.95269
1539:1428359163:var discipline=3 center=7 local_table=1 parmcat=192 parm=77:top of atmosphere:rpn_corr=0.997                             135:rpn_rms=0.454854
1540:1429989189:var discipline=3 center=7 local_table=1 parmcat=192 parm=78:top of atmosphere:rpn_corr=0.991                             974:rpn_rms=0.914833
1541:1431726116:var discipline=3 center=7 local_table=1 parmcat=192 parm=79:top of atmosphere:rpn_corr=0.989                             046:rpn_rms=1.35222
1542:1433579073:var discipline=3 center=7 local_table=1 parmcat=192 parm=80:top of atmosphere:rpn_corr=0.991                             348:rpn_rms=2.16879
1543:1435985994:var discipline=3 center=7 local_table=1 parmcat=192 parm=81:top of atmosphere:rpn_corr=0.998                             728:rpn_rms=0.686247
1544:1438131898:var discipline=3 center=7 local_table=1 parmcat=192 parm=82:top of atmosphere:rpn_corr=0.997                             273:rpn_rms=1.28427
1545:1440572657:var discipline=3 center=7 local_table=1 parmcat=192 parm=83:top of atmosphere:rpn_corr=0.993                             899:rpn_rms=1.94532
1546:1443031011:var discipline=3 center=7 local_table=1 parmcat=192 parm=84:top of atmosphere:rpn_corr=0.989                             526:rpn_rms=2.49445
1547:1445489387:var discipline=3 center=7 local_table=1 parmcat=192 parm=85:top of atmosphere:rpn_corr=0.989                             125:rpn_rms=1.99132
```
Please let me know if you see any unexpected changes. Thanks!

**Comment by EricJames-NOAA** [comment]
> @EricJames-NOAA I ran two tests with your branch and develop branch pn Hera.
> 
> ```
> your branch: /scratch1/NCEPDEV/stmp2/Wen.Meng/rrfs_2023030915-after
> develop branch: /scratch1/NCEPDEV/stmp2/Wen.Meng/rrfs_2023030915-before
> ```
> 
> I can see changes in simulated BT fields in NATLEV dataset between two tests:
> 
> ```
> 1529:1404731937:SBTA167:top of atmosphere:rpn_corr=0.831499:rpn_rms=8.65094
> 1530:1407483266:SBTA168:top of atmosphere:rpn_corr=0.995889:rpn_rms=0.503038
> 1531:1409196345:SBTA169:top of atmosphere:rpn_corr=0.991953:rpn_rms=0.924436
> 1532:1411015057:SBTA1610:top of atmosphere:rpn_corr=0.989674:rpn_rms=1.33921
> 1533:1413024108:SBTA1611:top of atmosphere:rpn_corr=0.992438:rpn_rms=2.07622
> 1534:1415614858:SBTA1612:top of atmosphere:rpn_corr=0.998844:rpn_rms=0.724649
> 1535:1417981775:SBTA1613:top of atmosphere:rpn_corr=0.997619:rpn_rms=1.21845
> 1536:1420618317:SBTA1614:top of atmosphere:rpn_corr=0.994744:rpn_rms=1.83284
> 1537:1423273914:SBTA1615:top of atmosphere:rpn_corr=0.990948:rpn_rms=2.36515
> 1538:1425904189:SBTA1616:top of atmosphere:rpn_corr=0.990229:rpn_rms=1.95269
> 1539:1428359163:var discipline=3 center=7 local_table=1 parmcat=192 parm=77:top of atmosphere:rpn_corr=0.997                             135:rpn_rms=0.454854
> 1540:1429989189:var discipline=3 center=7 local_table=1 parmcat=192 parm=78:top of atmosphere:rpn_corr=0.991                             974:rpn_rms=0.914833
> 1541:1431726116:var discipline=3 center=7 local_table=1 parmcat=192 parm=79:top of atmosphere:rpn_corr=0.989                             046:rpn_rms=1.35222
> 1542:1433579073:var discipline=3 center=7 local_table=1 parmcat=192 parm=80:top of atmosphere:rpn_corr=0.991                             348:rpn_rms=2.16879
> 1543:1435985994:var discipline=3 center=7 local_table=1 parmcat=192 parm=81:top of atmosphere:rpn_corr=0.998                             728:rpn_rms=0.686247
> 1544:1438131898:var discipline=3 center=7 local_table=1 parmcat=192 parm=82:top of atmosphere:rpn_corr=0.997                             273:rpn_rms=1.28427
> 1545:1440572657:var discipline=3 center=7 local_table=1 parmcat=192 parm=83:top of atmosphere:rpn_corr=0.993                             899:rpn_rms=1.94532
> 1546:1443031011:var discipline=3 center=7 local_table=1 parmcat=192 parm=84:top of atmosphere:rpn_corr=0.989                             526:rpn_rms=2.49445
> 1547:1445489387:var discipline=3 center=7 local_table=1 parmcat=192 parm=85:top of atmosphere:rpn_corr=0.989                             125:rpn_rms=1.99132
> ```
> 
> Please let me know if you see any unexpected changes. Thanks!

@WenMeng-NOAA those look like the expected changes!  I don't see anything unexpected.  Thanks!

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on WCOSS2, Hera and Orion. No baseline update is needed with this PR.

**Pull request merged**